Source/Scene/TileBoundingSphere.js

```diff
@@ -26,7 +26,7 @@ define([
     'use strict';
 
     var TileBoundingSphere = function(center, radius) {
```

Comment:
Since we changed all the other constructors to the newer format, might as well change this one too.